crates/ruff/src/rules/ruff/rules/snapshots/ruff__rules__ruff__rules__unreachable__tests__async-for.py.md.snap

```diff
@@ -0,0 +1,241 @@
+---
+source: crates/ruff/src/rules/ruff/rules/unreachable.rs
```

Comment:
I changed the file extension to `.md.snap`. This gives us a nice preview if you associate the file extension with markdown in your editor

![image](https://github.com/astral-sh/ruff/assets/1203881/9502eb76-2ed9-41a6-9883-a189d933f260)